web/packages/design/src/Card/Card.jsx

```diff
@@ -22,11 +22,11 @@ import { darkTheme } from './../theme';
 const Card = styled(Box)`
   box-shadow: ${props => props.theme.boxShadow[2]};
   border-radius: 8px;
+  background-color: ${props => props.theme.colors.levels.surface};
 `;
 
 Card.defaultProps = {
   theme: { darkTheme },
-  bg: 'levels.surface',
```

Comment:
This change removes the CSS exception we see in dev tools: 
<img width="272" alt="Screenshot 2023-06-12 at 10 24 37 AM" src="https://github.com/gravitational/teleport/assets/11967646/c92039eb-3011-4175-996f-b004c9b771d3">